flavor_test.go

```diff
@@ -121,3 +121,21 @@ func ExampleFlavor_Interpolate_sqlServer() {
 	// SELECT name FROM user WHERE id <> 1234 AND name = N'Charmy Liu' AND desc LIKE N'%mother\'s day%'
 	// <nil>
 }
+
+func ExampleFlavor_Interpolate_cql() {
+	sb := CQL.NewSelectBuilder()
+	sb.Select("name").From("user").Where(
+		sb.NE("id", 1234),
+		sb.E("name", "Charmy Liu"),
+		sb.Like("desc", "%mother's day%"),
+	)
+	sql, args := sb.Build()
+	query, err := CQL.Interpolate(sql, args)
+
+	fmt.Println(query)
+	fmt.Println(err)
+
+	// Output:
+	// SELECT name FROM user WHERE id <> 1234 AND name = 'Charmy Liu' AND desc LIKE '%mother''s day%'
```

Comment:
@SpencerC So, I'm curious about how these tests are being used.  I've seen a couple of these here which don't use valid CQL.  I don't know how important that is to the test itself, but CQL doesn't allow things like: OR, !=, JOIN, etc.  And it only allows LIKE queries under certain conditions.

Or is that not important since it won't really be running against Cassandra/Astra?

Reply:
In this particular instance, pkg.go.dev reads this as an example like this one: https://pkg.go.dev/github.com/huandu/go-sqlbuilder#example-Flavor.Interpolate-Sqlite

I will fix the CQL examples and tests to use valid CQL.

Reply:
Fixed.